src/components/SingleArticle.jsx

```diff
@@ -33,19 +36,49 @@ export default function SingleArticle(){
      if (loading) { return <div>Loading...</div>; } 
      if (error) { return <div>Error: {error.message}</div>; }
 
-    return( <div key ={article.article_id}> 
-    <div id= "singleArticle">
-    <p className = "articleItem"><strong>Title: </strong> {article.title}</p><div className= "container"><img src= {article.article_img_url} id= "articleImage"/></div>
-    <p className = "articleItem"><strong>Author: </strong> {article.author}</p>
-    <p className = "articleItem"><strong>Topic: </strong> {article.topic}</p>
+     function handleClick(event) {
+        event.preventDefault();
+        if (!showComments) {
```

Comment:
You could handle this by having the comments and the fetching of the data abstracted into a component